blankshield.js

```diff
@@ -45,27 +46,28 @@
    * @param {string} [strWindowName]
    * @param {string} [strWindowFeatures]
    */
-  blankshield.open = function(strUrl, strWindowName) {
+  blankshield.open = function(strUrl, strWindowName, strWindowFeatures) {
     var child;
     
-    if (strWindowName && strWindowName !== '_blank') {
-      open.apply(window, arguments);
+    if (strWindowName && (strWindowName === '_top' || strWindowName === '_self' || strWindowName === '_parent')) {
+      return open.apply(window, arguments);
     } else if (!oldIE) {
-      iframeOpen(strUrl);
+      return iframeOpen(strUrl, strWindowName, strWindowFeatures);
```

Comment:
I'm not sure that using the value returned here will be particularly useful, but it is consistent with the other branches